src/main/java/com/google/impactdashboard/database_manager/data_read/DataReadManagerImpl.java

```diff
@@ -1,10 +1,13 @@
 package com.google.impactdashboard.database_manager.data_read;
 
 import com.google.impactdashboard.data.project.ProjectIdentification;
+import com.google.impactdashboard.data.recommendation.IAMRecommenderMetadata;
 import com.google.impactdashboard.data.recommendation.Recommendation;
 import java.util.List;
+import java.util.Arrays;
 import java.util.Map;
-import java.lang.UnsupportedOperationException;
+import java.util.HashMap;
+import java.util.concurrent.atomic.AtomicReference;
 
 /** Class for returning data from the database. */
 public class DataReadManagerImpl implements DataReadManager {
```

Comment:
I thought this class read data from the real database. But seems like it's also returning fake data?

Reply:
In the prototype, the plan was to have only hardcoded data be passed back from the database backend. Eventually we will actually connect the database backend to the database, but not until the MVP.